docs/configuration.asciidoc

```diff
@@ -890,3 +890,11 @@ The unit is provided as suffix directly after the number, without any separation
  * `gb` (gigabytes)
 
 NOTE: we use the power-of-two sizing convention, e.g. `1 kilobyte == 1024 bytes`
+
+[float]
+[[special-configuration]]
+=== Special configuration
+
+Elastic APM patches `Kernel#require` to auto-detect and instrument supported third party libraries. It does so with the utmost care but for the purists this may be unacceptable.
```

Comment:
hmm saying that it might be unacceptable for purists sounds a tad snarky. How about something more neutral like "sometimes this conflicts with other libraries so we provide a setting to turn this functionality off"

Reply:
You're right 👍